docs/install-crossplane.md

```diff
@@ -49,6 +49,45 @@ For example:
 helm install --name crossplane --namespace crossplane-system crossplane-master/crossplane --version 0.0.0-249.637ccf9
 ```
 
+## Installing Cloud Provider Stacks
+
+After Crossplane has been installed, you can add additional functionality to its control plane by installing Crossplane Stacks.
```

Comment:
Somewhere in the example instructions we should link back to this heading..
"Install Crossplane (some existing link) **and  the GCP Cloud provider stack (link to this section)** ... "

I suppose it would be more hindrance than helpful if we included a StackRequest/StackInstall record in the example provider.yaml files, unless the StackRequest objects throughout the examples were identically named in the same namespace.